stdlib/ballerina-builtin/src/main/ballerina/ballerina/util/natives.bal

```diff
@@ -31,3 +31,13 @@ public native function base16ToBase64Encode (string baseString) (string);
 @Param {value:"s: string to be encoded"}
 @Return {value:"the encoded string."}
 public native function base64ToBase16Encode (string baseString) (string);
+
+@Description {value:"Encodes a base10 encoded string to url and filename safe base64."}
+@Param {value:"s: string to be encoded"}
```

Comment:
Shall we capitalize the first letter of the param description? i.e String to be encoded.

Reply:
I didn't change that. since that format is used in other places in this file.